board/src/main/java/com/hanhae99/board/controller/BoardController.java

```diff
@@ -0,0 +1,51 @@
+package com.hanhae99.board.controller;
+
+import com.hanhae99.board.domain.Board;
+import com.hanhae99.board.service.BoardService;
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.RequiredArgsConstructor;
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+import java.sql.Timestamp;
+import java.util.List;
+import java.util.stream.Collectors;
+
+@RestController
+@RequiredArgsConstructor
+public class BoardController {
+
+    private final BoardService boardService;
+
+
+    @GetMapping("/api/board")
```

Comment:
api의 네이밍만 봤을 땐 단건의 게시글을 조회하는 걸로 보입니다.
하지만 구현하시는 내용은 대량의 게시글 조회이기에 수정이 필요할 것 같아요.
restful하게 작성하실려면 자원의 경우 복수로 명시해주시는게 좋아요.

```suggestion
    @GetMapping("/api/boards")
```

그리고 만약 단 건의 조회의 경우엔`@GetMapping("/api/boards/{boardId}")` 이와 같이 작성해주시면 됩니다.